File: PHP/UberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car {
    public $typeCArAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCArAccepted, $seatsMaterial) {
        parent::__construct($license, $driver)
        $this->typeCArAccepted = $typeCArAccepted;
        $this->seatsMaterial = $seatsMaterial;
      }
}
?>

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, drive,typeCarAccepted,seatsMaterial):
        super.__init__(license, drive)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial